plugins/askrene/mcf.c

```diff
@@ -436,6 +436,16 @@ static struct arc node_adjacency_next(
 	return linear_network->node_adjacency_next_arc[arc.idx];
 }
 
+/* Set *capacity to value, up to *cap_on_capacity.  Reduce cap_on_capacity */
+static void set_capacity(s64 *capacity, u64 value, u64 *cap_on_capacity)
+{
+	*capacity = MIN(value, *cap_on_capacity);
+	if (*cap_on_capacity >= *capacity)
```

Comment:
This is always true, I bet 1 BTC.
That's why there was an assertion instead of an IF in the previous code.

Reply:
LoL, you're right!  I was thinking "of course cap can be less than value",  but completely missed the MIN().